readme.md

```diff
@@ -22,6 +22,12 @@ Promise.race([]);
 
 pRace([]);
 //=> [RangeError: Expected the input to contain at least one item]
+
+pRace(signal => [
```

Comment:
This should be documented in the API section with explanation of what `signal` is. Not everyone are familiar with it.

You also need to mention at the top of the readme that this package also adds abort abilities.